app/src/googlePlay/java/com/smlnskgmail/jaman/hashcheckerlite/MainActivity.java

```diff
@@ -7,6 +7,7 @@
 import android.content.IntentSender;
 import android.net.Uri;
 import android.os.Bundle;
+import android.util.Log;
```

Comment:
[Codacy](https://app.codacy.com/gh/hash-checker/hash-checker-lite/pullRequest?prid=7989232) find issue here. Please, fix it.

Reply:
done ✓